docs/data/base/components/modal/NestedModal.js

```diff
@@ -190,12 +190,11 @@ const TriggerButton = styled(Button)(
     border-color: ${theme.palette.mode === 'dark' ? grey[600] : grey[300]};
   }
   
-  &.active {
-    background-color: ${blue[700]};
-    box-shadow: none;
+  &:active {
+    background: ${theme.palette.mode === 'dark' ? grey[700] : grey[100]};
   }
   
-  &.focusVisible {
+  &:focus-visible {
```

Comment:
Was this supposed to use `Mui-focusVisible`, or is it intentional that we use the pseudo selector?

Reply:
Yep, I've been using `&:focus-visible` to keep things more generic and avoid tying it to Material-UI's specific class names. Just wanted a more neutral approach to styling the focus-visible state. Should we use `Mui-focusVisible` instead? tbh I don't know exactly what this would impact.

Reply:
The `Mui-focusVisible` is a workaround for older versions of Safari, that don't support `focus-visible` natively. According to [caniuse](https://caniuse.com/?search=focus-visible), these old Safari versions are used by about 1.7% of users, so I think it's safe to drop this workaround from our demos.